### PR TITLE
add the IAM role used for backups to outputs

### PR DESCRIPTION
**Pull request by dgoodellrhy**

### CODE REVIEW

**Review by sblack4 — APPROVED** [review]

**Review by rsanchez-s — APPROVED** [review]

File: outputs.tf [hunk 1]
```diff
@@ -3,3 +3,8 @@ output "efs_file_system_id" {
   description = "EFS File System ID"
   value       = try(aws_efs_file_system.this[0].id, "")
 }
+
+output "efs_backup_iam_role" {
+  description = "ARN of the IAM Role created to run AWS Backup"
+  value       = try(aws_iam_role.backup[0].id, "")
```
Comment (smiller171) [inline outputs.tf:9]: Would a null value not be better than an empty string if there's no IAM role?

```suggestion
  value       = try(aws_iam_role.backup[0].id, null)
```

**Review by smiller171 — COMMENTED** [review]
Looks like you have a stray `terraform-docs` file, and I think a null value would have been a better default...

File: outputs.tf (hunk 1 above)
Comment (dgoodellrhy) [reply outputs.tf:9]: the terraform-docs file was created and committed by the linter bot, I'm not.. super interested in troubleshooting our nannies at the moment, but I do agree on the null

Comment (smiller171) [reply outputs.tf:9]: ugh.

We've been disabling the linter bot slowly. I'll open an issue to disable it in this repo

**Pull request merged**